ertviz/controllers/parameter_selector_controller.py

```diff
@@ -0,0 +1,83 @@
+import regex
+from dash.exceptions import PreventUpdate
+from dash.dependencies import Input, Output, State
+from ertviz.models import (
+    load_ensemble,
+)
+
+old_submit = 0
+
+
+def filter_match(filter, key):
+    reg_exp = ".*" + ".*".join(filter.split())
+    try:
+        match = bool(regex.match(reg_exp, key))
+    except:
+        return False
+    return match
+
+
+def parameter_selector_controller(parent, app):
+    @app.callback(
+        [
+            Output(parent.uuid("parameter-selector-multi"), "options"),
+            Output(parent.uuid("parameter-selector-multi"), "value"),
+        ],
+        [
+            Input(parent.uuid("ensemble-selection-store"), "data"),
+            Input(parent.uuid("parameter-selector-filter"), "value"),
+            Input(parent.uuid("parameter-deactivator"), "value"),
+        ],
+    )
+    def update_parameters_options(selected_ensembles, filter_search, selected):
+        if not selected_ensembles:
+            raise PreventUpdate
+        selected = [] if not selected else selected
+
+        ensemble_id, _ = selected_ensembles.popitem()
+        ensemble = load_ensemble(parent, ensemble_id)
```

Comment:
Are parameter options equal for all ensembles, or do we want to make sure that all selected ensembles are considered when finding valid parameter options? 🤔 See #111 #108 #109

Reply:
Good comment! Will need to make it as an intersection of ensemble parameters.